sycl/include/sycl/ext/oneapi/experimental/sycl_complex.hpp

```diff
@@ -979,6 +979,35 @@ tan(const complex<_Tp> &__x) {
   return complex<_Tp>(__z.imag(), -__z.real());
 }
 
+// Literal suffix for complex number literals [complex.literals]
+inline namespace literals {
+inline namespace complex_literals {
+constexpr complex<sycl::half> operator""il(long double __im) {
```

Comment:
Do we really want `il` suffix to produce `complex<half>`? Looks a bit weird to me and it is not aligned with [C++ specification](https://eel.is/c++draft/complex.literals#:if). Would it be better to explicitly omit this specific literal, because `long double` is [not supported](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#subsec:scalartypes) by SYCL?

Reply:
I agree, I dropped the literal for `long double` to align with SYCL spec.